Number_types/include/CGAL/Sqrt_extension/Algebraic_extension_traits.h

```diff
@@ -22,7 +22,7 @@ template <class COEFF, class ROOT, class ACDE_TAG,class FP_TAG>
 class Algebraic_extension_traits<CGAL::Sqrt_extension<COEFF,ROOT,ACDE_TAG,FP_TAG> > {
 /* needed to 'add up' sqrt_extensions in iterator range such that all roots
    are collected in order to keep operation time minimal all scalar coeffs
-   are set to 1 by standardise.
+   are set to 1 by standardize.
```

Comment:
looks strange.  Maybe "by default"?

Reply:
It is only in a comment so let's just ignore it

Reply:
Indeed, changed it